tests/gplates_1_3/screen-output

```diff
@@ -38,7 +31,7 @@ Number of degrees of freedom: 10,656 (6,528+864+3,264)
    Solving Stokes system... 30+16 iterations.
 
    Postprocessing:
-     RMS, max velocity:                  0.0256 m/year, 0.102 m/year
+     RMS, max velocity:                  0.0257 m/year, 0.0769 m/year
```

Comment:
This is quite a change in max velocity. Before merging we need to check the new version is indeed more accurate than the old one.